C-Plus-Plus/stack/Check_for_balanced_parenthesis.cpp

```diff
@@ -1,72 +1,96 @@
 /*
 Check for balanced parenthesis
 ==============================
-Given an expression containing parenthesis, check if it is well-formed or balanced.
-Example of balanced parenthesis are: (), ((())), (a+b), (a/b)*(b/a)
-
+Given an expression containing parenthesis, check if it is well-formed or balanced. Abalanced parenthesis means for every opening bracket there must be equivalent closing 
+brackets.
```

Comment:
- Comment is too long kindly make it concise or split into sub-parts so that it will be readable.
- Use Multi line comments

Reply:
done